src/app/styles/globals.css

```diff
@@ -85,3 +85,7 @@ input[type="number"]::-ms-clear {
 [data-theme] {
   background-color: hsl(var(--b2) / var(--tw-bg-opacity, 1));
 }
+
+.custom-auto-row-grid {
+  grid-auto-rows: auto auto auto 1fr 1fr;
+}
```

Comment:
can those be replaced by a built-in tailwindcss class?

Reply:
i didn't find equivalent or custom arbitrary class for auto-grid-rows. let me know if you find any. i will update